### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

**Comment by Dmytry95** [comment]
image

**Comment by Dmytry95** [comment]
image

File: static/css/styles.css [hunk 1]
```diff
@@ -0,0 +1,7 @@
+body {
+    margin-top: 20px;
+}
+
+.sidebar {
+    padding: 0;
+}
```
Comment (vasylhnatiuk) [inline static/css/styles.css:7]: add newline

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]

**Review by vasylhnatiuk — APPROVED** [review]
Good job

**Pull request closed without merging**